compute_downstream.py

```diff
@@ -151,7 +151,8 @@ def main(args):
 
     # Compute connectivities if need to do cell type-level analysis
     if (len(CELLTYPE_LIST) > 0) & ("connectivities" not in adata.obsp):
-        sc.pp.neighbors(adata, n_neighbors=10, n_pcs=40)
+        sc.pp.pca(adata, n_comps=20)
```

Comment:
this is just to note that these are set as constants. (I guess for testing) and they will get removed anyway.

Reply:
Yes I wanted to make minimal change to old scripts while still making it work.